lib/ui/text.dart

```diff
@@ -1346,8 +1346,8 @@ class Paragraph extends NativeFieldWrapperClass2 {
   /// null, `ellipsis` was not null, and one of the lines exceeded the width
   /// constraint.
   ///
-  /// See the discussion of the `maxLines` and `ellipsis` arguments at [new
-  /// ParagraphStyle].
+  /// See the discussion of the `maxLines` and `ellipsis` arguments at
+  /// [new ParagraphStyle].
```

Comment:
ditto

Reply:
This one is ok, right?  I just moved new down to the next line.

Reply:
right. sorry